tests/libcxx/expected_results.txt

```diff
@@ -1068,6 +1069,9 @@ std/thread/thread.mutex/thread.mutex.requirements/thread.sharedtimedmutex.requir
 # Not analyzed, possible bug in uses_allocator_construction_args
 std/utilities/utility/mem.res/mem.poly.allocator.class/mem.poly.allocator.mem/construct_piecewise_pair_evil.pass.cpp FAIL
 
+# Not analyzed, likely STL bug.
+std/utilities/function.objects/func.wrap/func.wrap.func/noncopyable_return_type.pass.cpp SKIPPED

```

Comment:
This is a compiler bug. I've reported DevCom-10284753.

Reduced example ([Godbolt link]()):
```C++
template<class T>
struct Foo {};

template<class T>
void bar(T&) {}

template<class T>
void bar(const T&&) = delete;

int main()
{
    Foo<void()> x{};
    Foo<const void()> y{};
    bar(x);
    bar(y);
}
```